paloalto-cortex-xdr/release_notes.md

```diff
@@ -0,0 +1,7 @@
+#### What's Improved
+- Added for data ingestion support to pull Incidents from Palo Alto Cortex XDR.
```

Comment:
@cs-renuka-andhale please change this description. eg:
Added the ability to configure data ingestion (using the Data Ingestion Wizard). The Data Ingestion Wizard also supports multiple configurations specified on the Configurations tab of the Dragos WorldView Threat Intelligence connector, ensuring respective global variables based on the selected configuration are used while ingesting data.